kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html

```diff
@@ -308,22 +307,123 @@
 
             </dd>
 
+            <dt ng-hide="locationEditForm.$visible" class="image-info__wrap metadata-line metadata-line__key image-info__group--dl__key--panel">Location</dt>
+            <dd class="image-info__wrap metadata-line metadata-line__info image-info__group--dl__value--panel">
+                <button data-cy="it-edit-loc-button"
+                        class="image-info__edit"
+                        ng-if="ctrl.userCanEdit && ctrl.metadataUpdatedByTemplate.length == 0"
+                        ng-click="locationEditForm.$show()"
+                        ng-hide="locationEditForm.$visible">✎</button>
+
+                <span ng-hide="locationEditForm.$visible" ng-repeat="prop in ['subLocation', 'city', 'state', 'country']"
+                      >
+                        <span>
+                            <a
+                               ui-sref="{{ ctrl.hasMultipleValues(ctrl.rawMetadata[prop]) ? '' : 'search.results({query: (ctrl.metadata[prop] | queryFilter:ctrl.locationFieldMap[prop])})'}}"
+                               ng-class="{'image-info--multiple': ctrl.hasMultipleValues(ctrl.rawMetadata[prop]),
+                                          'inactiveLink': ctrl.hasMultipleValues(ctrl.rawMetadata[prop])}"
```

Comment:
This isn't a good way to disable a link - it's still partially accessible (it can be tabbed to and activated with the keyboard) and you need to forcibly override some of the link styling with !important. I'd suggest instead rendering an entirely different element (maybe a span?) if `hasMultipleValues` is true, which I think should also result in simpler elements

Reply:
Hi, @andrew-nowak  Thanks for the review, PR Updated